package.json

```diff
@@ -12,8 +12,7 @@
     ]
   },
   "scripts": {
-    "test": "npm run lint && npm run prepublish",
-    "test-unit": "browserify test/*.test.js | tap-closer | smokestack | tap-status",
+    "test": "tape -r ./test/mock-browser.js -r babel-register test/*.test.js",
```

Comment:
It's nice for making sure that PRs don't introduce linting problems to fail the tests on this